### ISSUE TITLE
Methods with empty names

### ISSUE THREAD

**Issue by sarna** [moonglum/halunke#20]
Hey, take a look at this
```
(Class new 'Foo
  attributes ["attribute"]
  methods @[
    "" { || (stdio puts "hello") }
  ]
)
('foo = (Foo new @["attribute" 0]))
(foo)
```
This prints "hello". Should it work this way?

**Comment by moonglum** [comment]
Uh, wow. That's interesting 🤔 That was definitely not intended. What do you think about it?

**Comment by sarna** [comment]
I've been thinking about possible use cases for this and 🤔 I don't see anything that it would be good for. Also, seems like messages with whitespace are allowed too ("say hello"), even a message with only a space - " ". I thought having a method with an empty name would block other methods from being called, but that's not the case - so it wouldn't be a security concern - I think 🤔
But is it worth it?

**Comment by moonglum** [comment]
I just [released Version 0.10](https://github.com/moonglum/halunke/releases/tag/v0.10.0). This release further improves the error handling in Halunke, with a focus on sending messages and defining methods. It is no longer possible to send an empty message and it is also not possible to define a method that expects an empty message name.

Thanks for looking into this ❤️
